packages/compass-components/package.json

```diff
@@ -34,6 +34,7 @@
     "@dnd-kit/core": "^6.0.7",
     "@dnd-kit/sortable": "^7.0.2",
     "@dnd-kit/utilities": "^3.2.1",
+    "@emotion/css": "^11.11.2",
```

Comment:
Why do we need this dependency?

Reply:
[Context for this](https://mongodb.slack.com/archives/G2L10JAV7/p1698163251582629?thread_ts=1698151766.108829&cid=G2L10JAV7)

Reply:
Ah cool, thanks. I wish we could leave a comment in json for on this one. Would it help if we add a comment in the `depcheck.rc`?

Reply:
sounds good, i'll follow up with that in a different PR, i have more random improvements coming up